libs/api/repository/src/lib/gn4/elasticsearch/elasticsearch.service.spec.ts

```diff
@@ -1,13 +1,18 @@
 import { ElasticsearchService } from './elasticsearch.service'
 import { ES_FIXTURE_AGGS_RESPONSE } from '@geonetwork-ui/common/fixtures'
-import { EsSearchParams } from '../types/elasticsearch.model'
+import { LangService } from '@geonetwork-ui/util/i18n'
+import { EsSearchParams } from '@geonetwork-ui/api/metadata-converter'
+
+const langServiceMock = {
+  iso3: 'eng',
+} as LangService
 
 describe('ElasticsearchService', () => {
   let service: ElasticsearchService
   let searchFilters
 
   beforeEach(() => {
-    service = new ElasticsearchService('fre')
+    service = new ElasticsearchService(langServiceMock, 'fre')
```

Comment:
Using a testBed here would make the test more readable I think, as it would be clear what are the injected dependencies (currently the test manipulates the service properties directly inc. private ones)